_posts/2017-06-07-unconf_projects_3.md

```diff
@@ -0,0 +1,66 @@
+---
+name: unconf17recap
+layout: post_discourse
+title: "Unconf projects 3: available, miner, rcheatsheet, ponyexpress"
+date: 2017-06-07
+authors:
+  - name: Karthik Ram
+categories: blog
+topicid: 740
+tags:
+- community
+- ropensci
+- meetings
+- unconf
+---
+
+Continuing our series of blog posts ([day 1](https://ropensci.org/blog/blog/2017/06/05/unconf_recap_1), [day 2](https://ropensci.org/blog/blog/2017/06/06/unconf_recap_2)) this week about [unconf 17](https://ropensci.org/blog/blog/2017/06/02/unconf2017). 
+
+
+
+## `available`
+
+**Summary:**  Ever have trouble naming your software package? Find a great name and realize it's already taken on CRAN, or further along in development on GitHub? The `available` package makes it easy to check for valid, available names, and also checks various sources for any unintended meanings. The package can also suggest names based on the description and title of your package. 
+
+![available Package](https://i.imgur.com/k6J5hQr.png)  
+
+**Team:** Jim Hester, Carl Ganz, Gábor Csárdi, Molly Lewis, Rachel Tatman
+
+**Github:** https://github.com/ropenscilabs/available
+
+
+## `miner`
+
+**Summary:**   
+The `miner` package provides a nice interface to the Minecraft API and allows users to manipulate the Minecraft world from a few simple functions. The package is designed with the intention of encouraging new users to learn R by writing scripts to automate activities inside Minecraft.
+
+[![Inserting ggplots into MineCraft](http://i.imgur.com/G0PComZ.png)](http://www.youtube.com/watch?v=zggCAYAfjXE "ggplot2 with Minecraft")
+
+**Team:** Brooke Anderson, Karl Broman, Gergely Daróczi, Mario Inchiosa, David Smith, Ali Zaidi 
+
+**Github:** https://github.com/ROpenSciLabs/miner
+
+
+## `rcheatsheet`
+
+**Summary:**  RStudio pioneered the idea of having beautiful cheatsheets for R packages. Creating them however, is quite time consuming and requires effort to maintain as packages evolve.  The `rcheatsheet` package makes it easy for any package author to quickly generate beautufil cheatsheets from a simple table stored on a Google sheet or a local table. This short slide deck provides a [quick overview](https://gitpitch.com/ramnathv/rcheatsheet/master?grs=github&t=moon)
+
+![R cheat sheet](http://i.imgur.com/JUC3Ljh.gif)
```

Comment:
this gif doesn't show up when rendered in browser, but does locally on my machine in Marked, probably ok